indico/modules/events/abstracts/models/abstracts.py

```diff
@@ -367,6 +374,18 @@ def candidate_tracks(self):
         return {t for t in self.reviewed_for_tracks if self.get_track_reviewing_state(t) in states}
 
     @property
+    def edit_track_mode(self):
+        if not inspect(self).persistent:
+            return EditTrackMode.both
+        elif self.state not in {AbstractState.submitted, AbstractState.withdrawn}:
+            return EditTrackMode.none
+        elif (self.public_state in (AbstractPublicState.awaiting, AbstractPublicState.withdrawn) and
+                self.reviewed_for_tracks == self.submitted_for_tracks):
```

Comment:
So when there is a disagreement between `reviewed_for_tracks` and `submitted_for_tracks` the submitter won't be able to change it even is there is no review? IIRC, last thing we discussed was that we would allow one to override another.